doc/release-notes-15996.md

```diff
@@ -0,0 +1,6 @@
+Deprecated or removed RPCs
+--------------------------
+
+- The `totalFee` option of the `bumpfee` RPC has been deprecated and will be
+  removed in 0.20.  To continue using this option start with
+  `-deprecatedrpc=totalFee`.  See the `bumpfee` RPC help text for more details.
```

Comment:
This is not going to be backported, so might as well inline it into the main release notes right now.

Reply:
Done.